lib/groonga/command/format/elasticsearch.rb

```diff
@@ -57,27 +59,27 @@ def json(options={})
                 }
               end
               components << JSON.generate(header)
+            when :columns
+              value.split(",").each do |column_name|
```

Comment:
We should use `split(/\s*,\s*)` because "_key, column" is a valid syntax.

BTW, why do you split the command object to name and arguments? `Groonga::Command::Load#columns` parses raw columns value.

Reply:
Just imitated the implementation of `Groonga:Command:Format:Command`, there is no more specific reason for split the command object to name and arguments.

Therefore, I use `Groonga::Command::Load`.